### ISSUE TITLE
Can we deprecate fatten arg of geom_pointrange()?

### ISSUE THREAD

**Issue by yutannihilation** [tidyverse/ggplot2#4881]
A follow up of #3672. Related to #4872, #2798

In my understanding, `fatten` was a workaround for the inability of controlling the point size and the line width differently. So, now that https://github.com/tidyverse/ggplot2/pull/3672 gets merged, I think `geom_pointrange()` no longer needs this parameter.

**Comment by yutannihilation** [comment]
Thinking this again, `geom_crossbar()` still needs `fatten` on the other hand because the middle bar is also line, not point. So, this seems a bit complicated than I thought. Anyway I think we need to clarify the relationship between `size`, `linewidth`, and `fatten`.

**Comment by thomasp85** [comment]
yeah, I think we may push this to the release after this in order to make the transition a bit smoother?

**Comment by yutannihilation** [comment]
If there's no downside, I think it's nice if we can include in v3.4.0. It might be another good example to show why separating `linewidth` from `size` is useful.
